auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -678,6 +678,12 @@ public PhoneBuilder setWhitelistedCountries(
                             "You can either whitelist or blacklist country codes for phone " +
                                     "authentication.");
                 }
+
+                String message = "Invalid argument: The whitelist is %s. For this case, " +
```

Comment:
Instead of: "For this case, don't use this method" how about  something like "Only non-null and non-empty whitelists are valid. To specify no whitelist, do not call this method."

Reply:
Much better. :+1: